### PR TITLE
refactor: decommission `@nhost/core`

### PR DESCRIPTION
**Pull request by plmercereau**
Move the `@nhost/core` to `@nhost/hasura-auth-js` and `@nhost/hasura-storage-js`

### CODE REVIEW

File: config/tsconfig.base.json
Comment (szilarddoro) [inline config/tsconfig.base.json:31]: Have you changed something in your formatter configuration? It's not the same as the output my code editor has.

Comment (plmercereau) [reply config/tsconfig.base.json:31]: no, I haven't, but talking about that, the `source.organizeImports` vscode option conflicts with the lint sort rule
I think we should disable this lint rule once and for all...

Comment (szilarddoro) [reply config/tsconfig.base.json:31]: I support this idea 😄

**Review by szilarddoro — APPROVED** [review]

**Comment by szilarddoro** [comment]
I think we should wait for someone else's approval on this PR.

**Review by nunopato — COMMENTED** [review]
Is this a major version bump? core's functionality is being moved elsewhere but the API isn't changing, is it?

**Pull request merged**
